spec/lib/classroom/scopes_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Classroom::Scopes do
```

Comment:
Yeah, this is good enough to jog our memory if things need to be changed in the future.